_posts/2018-03-29-new-profiles.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: New Profiles
+date: 2018-03-29
+categories:
+  - projects
+description: New CSH profiles in Flask because the old one broke
+image: https://i.imgur.com/KGNp3xX.png
+image-sm: https://i.imgur.com/KGNp3xX.png
+author: Nate Dooley
+author-image: https://profiles.csh.rit.edu/image/dooley
+author-bio: Freshman CIT major
+author-email: dooley@csh.rit.edu
+author-social:
+  github: https://github.com/natedooley
+---
+
```

Comment:
Add a section header "What is it?" or similar.

Reply:
yes